.gitmodules

```diff
@@ -3,7 +3,8 @@
 	url = https://github.com/ceph/ceph-object-corpus.git
 [submodule "src/civetweb"]
 	path = src/civetweb
-	url = https://github.com/ceph/civetweb
+	url = https://github.com/mdw-at-linuxbox/civetweb
+	branch = wip-listen2
```

Comment:
@mdw-at-linuxbox please drop this change. Messing with submodule urls is one of the better ways to annoy everyone. We can create a branch on ceph/civetweb instead